pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -1214,7 +1215,7 @@ def dataType(obj):
             elif obj.ndim == 2 and obj.dtype.names is None and obj.shape[1] == 2:
                 return 'Nx2array'
             else:
-                raise Exception('array shape must be (N,) or (N,2); got %s instead' % str(obj.shape))
+                raise ArrayDimensionError('array shape must be (N,) or (N,2); got %s instead' % str(obj.shape))
```

Comment:
I do think it's a good idea to introduce more specific exceptions for this sort of thing, but I think for this purpose, we should probably stick with `ValueError` which I believe is what numpy raises when you get a dimension mismatch related exceptions.

Reply:
Thank you for the review @j9ac9k , will update it to `ValueError`.